src/scss/_mixins.scss

```diff
@@ -82,7 +82,11 @@
 	// set default foreground and background colour
 	color: $foreground-color;
 	background-color: $background-color;
-	text-align: _oMessageGet('text-align', $from: $opts);
+	text-align: _oMessageGet('align', $from: $opts);
+
+	.o-message__container {
+		justify-content: _oMessageGet('align', $from: $opts);
```

Comment:
Just to confirm, for IE 11 it will look like it currently does right? Because IE 11 does not support `justify-content`

Reply:
It does :) There are [plenty of bugs to watch out for though](https://github.com/philipwalton/flexbugs) so it was worth double checking:
![Screenshot 2021-03-16 at 11 31 41](https://user-images.githubusercontent.com/10405691/111302351-354f7100-864b-11eb-9ede-cb00459dcc6f.png)

Reply:
Oh, caniuse stated it had no support. Glad to know it looks good 👍